fdbclient/BlobCipher.cpp

```diff
@@ -465,8 +520,10 @@ EncryptBlobCipherAes265Ctr::EncryptBlobCipherAes265Ctr(Reference<BlobCipherKey>
 
 EncryptBlobCipherAes265Ctr::EncryptBlobCipherAes265Ctr(Reference<BlobCipherKey> tCipherKey,
                                                        Reference<BlobCipherKey> hCipherKey,
-                                                       const EncryptAuthTokenMode mode)
-  : ctx(EVP_CIPHER_CTX_new()), textCipherKey(tCipherKey), headerCipherKey(hCipherKey), authTokenMode(mode) {
+                                                       const EncryptAuthTokenMode mode,
+                                                       BlobCipherMetrics::UsageType usageType)
+  : ctx(EVP_CIPHER_CTX_new()), textCipherKey(tCipherKey), headerCipherKey(hCipherKey), authTokenMode(mode),
+    usageType(usageType) {
 	ASSERT(isEncryptHeaderAuthTokenModeValid(mode));
```

Comment:
please add `assert` ensuring validation of `type` input parameter.

Reply:
ditto